### PR TITLE
Test PR for checking status of fork - DO NOT MERGE

### PR DESCRIPTION
**Pull request by labkode**
This issue keeps track of the fork state in cernbox/web

### CODE REVIEW

File: packages/web-app-files/src/services/folder/loaderSharedWithOthers.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ export class FolderLoaderSharedWithOthers implements FolderLoader {
 
       resources = yield resources.json()
       resources = resources.ocs.data
+      // filter out shares via link only
+      resources = resources.filter((r) => r.share_type === 0)
```
Comment (pascalwengerter) [inline packages/web-app-files/src/services/folder/loaderSharedWithOthers.ts:34]: @diocas you can omit this commit, we have https://github.com/owncloud/web/blob/7e8e94964c2b51470459db0b9aa13515f8aad052/packages/web-app-files/src/services/folder/loaderSharedWithOthers.ts#L34 on master now which doesn't even include link shares in the request :)

Comment (diocas) [reply packages/web-app-files/src/services/folder/loaderSharedWithOthers.ts:34]: Perfect, updated!

File: packages/web-app-files/src/helpers/resources.js [hunk 2]
```diff
@@ -273,13 +273,18 @@ export function aggregateResourceShares(
   token
 ) {
   if (incomingShares) {
+    shares = addSharedWithToShares(shares)[0]
```
Comment (kulmann) [inline packages/web-app-files/src/helpers/resources.js:283]: This commit here can be omitted now. https://github.com/owncloud/web/pull/6717 got merged.

**Comment by pascalwengerter** [comment]
@labkode @elizavetaRa can this be closed? Last activity has been 5 months ago, and the PR is over a year old now

**Comment by kulmann** [comment]
for `checking status of fork` 😉 which does make sense.

**Comment by labkode** [comment]
@diocas I keep it open to understand the divergence of our work :)

**Comment by diocas** [comment]
We no longer use the qa branch... It's `cernbox` for current prod (which is very old by now), but then we have our dev branches where we're using today's master (i.e `diogo` branch usually is the most up to date).

**Comment by labkode** [comment]
Superseded by https://github.com/owncloud/web/pull/9094

**Pull request closed without merging**